packages/shared/src/components/cards/PostButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react';
+import { CardButton } from './Card';
+
+export type PostButtonProps = {
+  title: string;
+  onClick?: (e?: React.MouseEvent) => unknown;
+};
+
+export default function PostButton({
+  title,
+  onClick,
+}: PostButtonProps): ReactElement {
+  return (
+    <CardButton
+      title={title}
+      onClick={onClick}
+      onMouseUp={(event) => event.button === 1 && onClick?.(event)}
```

Comment:
If it is used only for real buttons, we should not support middle click. The middle click here supports links